examples/ct/handlers.go

```diff
@@ -337,20 +338,20 @@ func getSTH(c LogContext, w http.ResponseWriter, r *http.Request) (int, error) {
 	}
 
 	// Now build the final result object that will be marshalled to JSON
-	jsonResponse := getSTHResponse{
+	jsonRsp := getSTHResponse{
```

Comment:
I don't think it's a big deal but I always use resp so one of us is going to be unhappy.